app/models/course/assessment/question/forum_post_response.rb

```diff
@@ -9,6 +9,20 @@ def question_type
     I18n.t('course.assessment.question.forum_post_responses.question_type')
   end
 
+  def attempt(submission, last_attempt = nil)
+    answer =
+      Course::Assessment::Answer::ForumPostResponse.new(submission: submission, question: question)
+    last_attempt&.answer_options&.each do |answer_option|
```

Comment:
Can I check what `answer_options` here is?

Reply:
will be fixing this in Part 4